docs/sources/mimir/manage/mimir-runbooks/_index.md

```diff
@@ -1411,6 +1411,23 @@ How to **investigate** and **fix** it:
     - Check the number of in-memory series shown on the `Mimir / Tenants` dashboard for an approximation of the number of series that will be compacted once these blocks are shipped from ingesters.
     - Check the configured `compactor_split_and_merge_shards` for the tenant. A reasonable rule of thumb is 8-10 million series per compactor shard - if the number of series per shard is above this range, increase `compactor_split_and_merge_shards` for the affected tenant(s) accordingly.
 
+##### Persistent Volume Resizing Prerequisite - jsonnet
+
+If persistent volume resizing is required for store-gateways and automatic downscaling is enabled, it must be disabled before proceeding with the resizing process. This step is necessary to prevent any unexpected downscaling by the rollout operator while updating the stateful set for each zone.
```

Comment:
```suggestion
If persistent volume resizing is required for store-gateways and automatic downscaling is enabled, you must disable it before proceeding with the resizing process. This step is necessary to prevent any unexpected downscaling by the rollout operator while updating the stateful set for each zone.
```

Reply:
We try to use active voice whenever possible, per https://grafana.com/docs/writers-toolkit/write/style-guide/style-conventions/#write-in-active-voice.